docs/usage.md

```diff
@@ -497,6 +497,87 @@ CodeChecker cmd diff --basename ./reports --newname ./reports-ctu --new
  You can also use JSON format output of `CodeChecker cmd diff` command if
  you want to use it for further processing by an other program.
 
+### Run comparison
```

Comment:
The content of this whole section could be moved under https://github.com/Ericsson/codechecker/blob/607c27991aa8173a21e3bc440e1adabb6c038a7b/docs/usage.md#diffs-between-runs

because it is about the same topic, but more detailed and with example.

And you could refer to this section at the end of https://github.com/Ericsson/codechecker/blob/607c27991aa8173a21e3bc440e1adabb6c038a7b/docs/usage.md#using-diff